packages/api/src/external/aws/app-config.ts

```diff
@@ -170,6 +174,11 @@ export async function isIHEGatewayV2EnabledForCx(cxId: string): Promise<boolean>
   return cxIdsWithIHEGatewayV2Enabled.some(i => i === cxId);
 }
 
+export async function isEpicEnabledForCx(cxId: string): Promise<boolean> {
+  const cxIdsWithEpicEnabled = await getCxsWithEpicEnabled();
```

Comment:
if the array is empty - let's return `true`, otherwise there may be slight downtime with enabled cxs during deployment

Reply:
good catch!